packages/datetime/test/dateRangePickerTests.tsx

```diff
@@ -110,6 +110,15 @@ describe("<DateRangePicker>", () => {
             assert.equal(dateRangePicker.state.displayMonth, NOVEMBER);
         });
 
+        it("is value - 1 if set and initialMonth not set and value month === maxDate month", () => {
```

Comment:
I thought the same thing about the wording in some of these tests. The problem is that the issue is tied to a specific setup when using the API, which in effect leads to describing the code.